### ISSUE TITLE
Formatting removes parentheses and breaks compiled attribute

### ISSUE THREAD

**Issue by dominikb** [ThisIsManta/vscode-stylus-supremacy#25]
# Issue
I am using a property lookup to style a text-shadow. Formatting the file removes the parentheses and adds a whitespace.  
Thus the stylus compiler interprets the line as expression and subtracts both values, resulting in an invalid CSS property.

# Expected Result
The parentheses do not get removed as this changes the outcome.  
The expression should result in `-4rem`.

# VSCode Extension Configuration
````javascript
{
  // ...
  "stylusSupremacy.insertBraces": false,
  "stylusSupremacy.insertLeadingZeroBeforeFraction": false,
  "stylusSupremacy.insertColons": false,
  "stylusSupremacy.insertSemicolons": false,
  "stylusSupremacy.sortProperties": "alphabetical",
  "stylusSupremacy.selectorSeparator": ","
}
````

# Example
````stylus
h1
  font-size 4rem
  text-shadow 2rem (-@font-size) #fff
````

becomes

````stylus
h1
  font-size 4rem
  text-shadow 2rem - @font-size #fff
````
## Compiled CSS Ouput
__Expected__
````css
h1 {
text-shadow: 2rem -4rem #fff;
}
````
__Actual__
````css
h1 {
text-shadow: -2rem #fff;
}
````

**Comment by ThisIsManta** [comment]
Thank you for reporting this issue. Please spare me some time to fix.

**Comment by ThisIsManta** [comment]
Fixed at [v2.12.5](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.12.5)

Cheers!
